src/esp/bindings/PhysicsObjectBindings.cpp

```diff
@@ -187,9 +201,14 @@ void declareBasePhysicsObjectWrapper(py::module& m,
       .def_property_readonly(
           "user_attributes", &PhysObjWrapper::getUserAttributes,
           ("User-defined " + objType +
-           " attributes.  These are not used internally by Habitat in any "
+           " attributes. These are not used internally by Habitat in any "
            "capacity, but are available for a user to consume how they wish.")
               .c_str())
+      .def_property_readonly(
+          "marker_sets", &PhysObjWrapper::getMarkerSets,
+          py::return_value_policy::reference_internal,
+          ("The MarkerSets defined for " + objType + " this object.").c_str())
```

Comment:
```suggestion
      .def_property_readonly(
          "marker_sets", &PhysObjWrapper::getMarkerSets,
          py::return_value_policy::reference_internal,
          ("Editable reference to the MarkerSets defined for " + objType + " this object.").c_str())
```
This is an editable reference, right? Or do I need to re-register it after edits?

Reply:
Well, it is an editable reference, but it is of a copy of the Configuration (same as the user_defined attributes) so it will require re-registering of the base attributes on edits.

Reply:
Ok, so my edits will stick around within the object, but they won't be written into any configuration unless I ask for it?